src/main/angular/app/error_page/error_page.component.html

```diff
@@ -0,0 +1,8 @@
+<div id="error-container">
+  <div id="errors">
+    <div class="error" *ngFor="let error of errors">
+      <h3>{{ error.message }}</h3>
+      <p>{{ error.exception }}</p>
+    </div>
+  </div>
```

Comment:
In general, it's not a good practice to show all the error message details to users for the following reasons:
1. For users that don't understand the messages, it can be overwhelming. 
2. For users that can understand the messages, this represents a security vulnerability because they learn how the systems are working.  


Instead, you should be able to determine the kind of error based upon the HTTP response code. If so inclined, you may show them some details (on a permission denied, tell them that). Otherwise, a generic error message is best.

Reply:
At the moment, it's difficult to tell what is sent back when something like permissions fail. 

For now, I've just included the message for ease-of-use, but in the future I will get together with Ionis to figure out possible responses. Is that OK or would you like me to say something vague like "An error occured. Please check the server logs for details"?

Reply:
For now, a generic message would be best. 

In terms of contents of that message, it should be user oriented, something simple like "An error occurred" is sufficient. Something analogous to the Twitter Fail Whale

Reply:
Got it. I'll put off a super playful error page for later, though :)